src/Routes.js

```diff
@@ -62,6 +63,7 @@ const Routes = (props) => (
     <Route path="/data_sharing_language_tool" component={DataSharingLanguageTool} />
     <Route path="/nih_dms_policy" component={NIHDMSPolicyInfo} />
     <Route path="/anvil_dms_policy" component={AnVILDMSPolicyInfo} />
+    <AuthenticatedRoute path="/test_user_profile" component={UserProfile} props={props} rolesAllowed={[USER_ROLES.all]} />
```

Comment:
nit: I think we can simplify this to `user_profile` which won't require a path change once this is the official version of the page.